### ISSUE TITLE
Is there anyway to see the label or visualise what X_TEST? 

### ISSUE THREAD

**Issue by Symbadian** [kcct-fujimotolab/3DCNN#25]
Hi @kcct-fujimotolab,
I have been trying assiduously to see the type of data that is passed in via the X-Test variable.

Can you guide me on this, please?
for example
  ```X_train, X_test, Y_train, Y_test = train_test_split(
        X, Y, train_size=0.8, test_size=0.2, random_state=1) `` `
        
        How can we see what videos are there in this X-Test variable?
        I get the output from the predictions but they are all numeric like this:

  ```predict_labels = model.predict(X_test, batch_size=args.batch,verbose=1)
 print('This is prediction labels',predict_labels) ```

 ``` 
 This is my output her from the predictions but there is no way to tell what videos are being processed????!!!
[0.55468273 0.44531724]
 [0.56695396 0.43304604]
 [0.5598244  0.4401756 ]
 [0.60472274 0.39527726]
 [0.5598211  0.44017896]
etc..
 ```
 
 how can I see the videos that are being referenced here?
 thank you in advance for acknowledging my challenge. 
 Any guidance to a link or explanations would be very helpful.
